apps/spruce/src/pages/adminSettings/tabs/AnnouncementsTab/AnnouncementTab.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from "@emotion/styled";
+import { H2, H3 } from "@leafygreen-ui/typography";
+
+export const AnnouncementTab = () => (
+  <div>
+    <TitleContainer>
+      <H2 data-cy="preferences-tab-title">Announcements</H2>
+    </TitleContainer>
+    <H3>Announcements</H3>
+    <p>Announcements settings for the application.</p>
+  </div>
+);
+
+const TitleContainer = styled.div`
+  margin-bottom: 30px;
```

Comment:
nit. use tokens where possible i.e.

```
import { size } from "@evg-ui/lib/constants/tokens";
```

```suggestion
  margin-bottom: ${size.m};
```